specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/CommonDefinitions.json

```diff
@@ -255,6 +255,31 @@
         }
       }
     },
+    "ExtendedLocation": {
```

Comment:
Is there already a definition for this in the common-types?

Reply:
currently no